tests/integration/conftest.py

```diff
@@ -101,7 +104,10 @@ def factory(
             c2r_runtime.expect(pexpect.EOF)
             c2r_runtime.close()
         finally:
-            shell("subscription-manager unregister")
+            if unregister:
+                shell("subscription-manager unregister")
+            else:
+                pass
```

Comment:
This else statement is not actually necessary. If the `unregister` variable is set to `False`, the function flow will continue to the `return factory` either way.

```suggestion
```

Reply:
I see, not sure why I left that there :D